docs/cloud/aws/lca1/billing/budget-actions.md

```diff
@@ -0,0 +1,96 @@
+# Configuring a budget action
+
+You can attach budget actions to an alert for either a cost budget or a usage budget. To configure a budget action on a new budget, first follow the steps for [Creating a cost budget](https://docs.aws.amazon.com/cost-management/latest/userguide/create-cost-budget.html)
+
+## Creating a budget
+
+1. Sign in to the AWS Management Console and open the [AWS Cost Management console](https://docs.aws.amazon.com/cost-management/latest/userguide/create-cost-budget.html)
+
+2. In the navigation pane, choose **Budgets**. (You may directly search for "budgets" in top search bar)
+
+3. Click on "create budget", choose Budget setup as "Customize(advanced)" and Budget type as "Cost budget"
```

Comment:
Can we standardize on either quotes or bolded text?

Reply:
Switched to bold in a2df5681b85c5e3a9549654c3b100d67b3c6df13. I've left the single search term in quotes because it's a different use case.